Cesium3DTiles/src/TileID.cpp

```diff
@@ -40,4 +42,58 @@ TileIdUtilities::createTileIdString(const TileID& tileId) {
   return std::visit(Operation{}, tileId);
 }
 
+/*static*/ std::optional<TileID>
+TileIdUtilities::getParentTileID(const TileID& tileId) {
+  struct Operation {
+
+    std::optional<TileID> operator()(const uint32_t& /*id*/) {
+      return std::nullopt;
+    }
+
+    std::optional<TileID> operator()(const std::string& /*url*/) {
+      return std::nullopt;
```

Comment:
this feels somehow awkward as one would expect this method to return the url of the parent